packages/supabase/lib/src/realtime_client_options.dart

```diff
@@ -1,5 +1,6 @@
 /// {@template realtime_client_options}
-/// Options to pass to the RealtimeClient
+/// Options to pass to the RealtimeClient.
+/// Default is 10 per second
```

Comment:
Thanks for this! Maybe we should add the note about the default value on `eventsPerSecond` property instead of on `RealtimeClientOptions` itself. 

Also now that you mention it, maybe we can clarify the wording on the comments of `eventsPerSecond`. Do you think something like "How many broadcast events the RealtimeClient can push in a second" would be better?